xml/snmp_sysdescr.xml

```diff
@@ -6218,6 +6218,35 @@ Copyright (c) 1995-2005 by Cisco Systems
   <!--======================================================================
                               XEROX
    =======================================================================-->
+  <fingerprint pattern="^Xerox Phaser (\d+)MFP;\sOS\s(\d+\.\d+\.\d+\.\d+).*$" certainty="1.0">
+    <description>Xerox Phaser 3000 MFP Series </description>
+    <example hw.version="3300" os.version="1.50.00.17">Xerox Phaser 3300MFP; OS 1.50.00.17   05-07-2013, Engine 1.05.44, NIC V4.03.01(P3300MFP), PCL5e 6.08.01 11-10-2009, PCL6 5.98  09-23-2009, PS3 V2.19.06 12-15-2010, SPL 5.24 03-27-2006, PDF V1.00.32 02-25-2006, IBM/EPSON 5.22 09-24-2009</example>
+    <example hw.version="3300" os.version="1.50.00.14">Xerox Phaser 3300MFP; OS 1.50.00.14   07-16-2009, Engine 1.05.44, NIC V4.02.06(P3300MFP) 07-16-2009, PCL5e 5.93 03-19-2009, PCL6 5.94  05-11-2009, PS3 V1.99.06 04-09-2009, SPL 5.24 03-27-2006, PDF V1.00.32 02-25-2006, IBM/EPSON 5.20 02-03-2009</example>
+    <param pos="0" name="os.vendor" value="Xerox"/>
```

Comment:
Here and below, add `hw.vendor, ,etc as we know the hardware vendor, family, and device.

Reply:
Good call, have added `hw.vendor`, `hw.device` and `hw.family` to all Xerox fingerprints that were missing those parameters